File: classes/en/Calificacion.php
<?php
class EN_Calificacion{

var $idccalificacion;
var $usuario;
var $utilidad;
var $facilidad;
var $estilo;
var $soporte;
var $notas;

function setIdccalificacion($idccalificacion) { $this->idccalificacion = $idccalificacion; }
function getIdccalificacion() { return $this->idccalificacion; }
function setUsuario($usuario) { $this->usuario = $usuario; }
function getUsuario() { return $this->usuario; }
function setUtilidad($utilidad) { $this->utilidad = $utilidad; }
function getUtilidad() { return $this->utilidad; }
function setFacilidad($facilidad) { $this->facilidad = $facilidad; }
function getFacilidad() { return $this->facilidad; }
function setEstilo($estilo) { $this->estilo = $estilo; }
function getEstilo() { return $this->estilo; }
function setSoporte($soporte) { $this->soporte = $soporte; }
function getSoporte() { return $this->soporte; }
function setNotas($notas) { $this->notas = $notas; }
function getNotas() { return $this->notas; }
function getNombreUsuario(){
  return $this->usuario->getUser();	
}
	
			function generarCalificacion($json2){
				echo $json2.'es la cadena';
				$json=json_decode($json2,true);
			 $this->setIdccalificacion($json['idccalificacion']);
				
			 $useraux=new EN_Usuario();
			$useraux->setIdUsuario($json['usuario']['user']);

		    $useraux->setUser($json['usuario']['user']);
			 
			 $this->usuario=$useraux;
				
			 $this->setUtilidad($json['utilidad']);
			 $this->setFacilidad($json['facilidad']);
			 $this->setEstilo($json['estilo']);
			 $this->setSoporte($json['soporte']);
			 $this->setNotas($json['notas']);
			echo json_encode($this);
		}

}

?>

File: Android/RapiMoncha/src/una/android/rapimoncha/entidades/Calificacion.java
package una.android.rapimoncha.entidades;

import org.json.JSONObject;

import una.android.rapimoncha.interfaces.IJsonable;

public class Calificacion implements IJsonable{
	private int idclasificacion;
	private Usuario usuario;
	private int utilidad;
	private int facilidad;
	private int estilo;
	private int soporte;
	private String notas;
	
	public int getIdclasificacion() {
		return idclasificacion;
	}
	public void setIdclasificacion(int idclasificacion) {
		this.idclasificacion = idclasificacion;
	}
	public Usuario getUsuario() {
		return usuario;
	}
	public void setUsuario(Usuario usuario) {
		this.usuario = usuario;
	}
	public int getUtilidad() {
		return utilidad;
	}
	public void setUtilidad(int utilidad) {
		this.utilidad = utilidad;
	}
	public int getFacilidad() {
		return facilidad;
	}
	public void setFacilidad(int facilidad) {
		this.facilidad = facilidad;
	}
	public int getEstilo() {
		return estilo;
	}
	public void setEstilo(int estilo) {
		this.estilo = estilo;
	}
	public int getSoporte() {
		return soporte;
	}
	public void setSoporte(int soporte) {
		this.soporte = soporte;
	}
	public String getNotas() {
		return notas;
	}
	public void setNotas(String notas) {
		this.notas = notas;
	}
	
    @Override
    public JSONObject getJson() {
        JSONObject obaux=new JSONObject();
        try {
        	obaux.put("idccalificacion", idclasificacion);
            obaux.put("usuario", usuario.getJson());
            obaux.put("utilidad",utilidad );
            obaux.put("facilidad",facilidad );
            obaux.put("estilo",estilo );
            obaux.put("soporte", soporte);
            obaux.put("notas",notas );
        }catch(Exception ex){
            ex.printStackTrace();
        }
        return obaux;
    }


}
